src/Discord.py

```diff
@@ -0,0 +1,69 @@
+import requests, json
+from src.Dev import Dev
+from src.Settings import Settings
+from src.TERMGUI.Log import Log
+
+from discord_webhook import DiscordWebhook
+
+
+class Discord:
+
+    dev_endpoint = Settings.get_key(Settings.discord_dev_key)
+    prod_endpoint = Settings.get_key(Settings.discord_prod_key)
+
+    def __init__(self, content = "", quiet = False):
```

Comment:
Since you changed the way we interact with this class, I don't think we need the content argument here anymore?

Reply:
No, it's not necessary. I was kind of thinking it would be better if we were passing all the text to the constructor but it's not practical. I'll remove it for now.